docs/dev/libs/state_processor_api.md

```diff
@@ -345,6 +345,44 @@ Along with reading registered state values, each key has access to a `Context` w
 
 {% panel **Note:** When using a `KeyedStateReaderFunction`, all state descriptors must be registered eagerly inside of open. Any attempt to call a `RuntimeContext#get*State` will result in a `RuntimeException`. %}
 
+### Window State
+
+The state processor api supports reading state from a [window operator]({{ site.baseurl }}/dev/stream/operators/windows.html).
+This includes both time based windows along with other types, pre-aggregation, non-aggregated, and windows with evictors.
```

Comment:
This ("along with other types") is vague. I don't see code for handling count windows, or for handling state in Triggers -- but there's a lot of code here, so maybe I missed something. Can we be more precise about what is and isn't supported?

Reply:
I'll try and clarify. Count windows do work. The timers set via triggers can be read via the WindowReader#Context, other state is not currently supported.